docs/user_guide/disconnected/index.md

```diff
@@ -4,286 +4,52 @@ title: Disconnected
 sidebar: sidebars/documentation.html
 ---
 
-# Katello Disconnected Utility
+# Katello Inter-Server Sync
 
 ## Intro
 
-If you are working in an environment where you wish to synchronize content from Red Hat's CDN and make available to your Katello servers that do not have a connection to the internet we have a utility that assists with this, known as katello-disconnected.  A diagram of the content flow is as follows:
+If you are working in an air-gapped network environment where some of your
+Katello servers do not have Internet connectivity, you may be interested in
+using the Katello Inter-Server Sync feature. This allows you to export repos
+(including repos in content views) on your "upstream" Katello server, and then
+import said repos into your "downstream" server that does not have
+connectivity.
 
-![Disconnected Workflow](./katello-disconnected.png)
+It is possible to export either individual yum repos, or all yum repos in a
+particular content view. For now, only yum content is supported. Future
```

Comment:
You can avoid passive voice by swapping things around:

"For a content view, individual or all yum repos can be exported."